packages/svelte/package.json

```diff
@@ -2,45 +2,48 @@
   "name": "@tolgee/svelte",
   "version": "4.8.0",
   "scripts": {
-    "build": "svelte-kit package",
+    "dev": "vite dev",
+    "build": "svelte-kit sync && svelte-package",
+    "develop": "sleep 5 && nodemon --exec svelte-package",
     "clean": "rm -rf build build-e2e dist dist-e2e lib .svelte-kit",
-    "preview": "svelte-kit preview",
```

Comment:
Why to remove this?

Reply:
There is no app since this is a library, so nothing to preview. I guess it may as well sit and wait there.

Reply:
added back